README.md

```diff
@@ -85,6 +85,20 @@ Make sure to update the following metadata fields:
 * `maintainer`
 * `description`
 
+## Adding provider documentation
```

Comment:
@tomkerkhove et all: should this title have "authentication" somewhere in it? Maybe:

```suggestion
## Adding authentication-provider documentation
```

Or:

```suggestion
## Adding documentation for an authentication provider
```

Reply:
Yes, I'd suggest to go with this to follow the scaler approach
```suggestion
## Writing documentation for a new authentication provider
```

Reply:
Done! 👍